shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -85,6 +85,29 @@
   private static final float SCROLL_EXTENT_FOR_INFINITY = 100000.0f;
   private static final float SCROLL_POSITION_CAP_FOR_INFINITY = 70000.0f;
   private static final int ROOT_NODE_ID = 0;
+  private static final int SCROLLABLE_ACTIONS =
+      Action.SCROLL_RIGHT.value
+          | Action.SCROLL_LEFT.value
+          | Action.SCROLL_UP.value
+          | Action.SCROLL_DOWN.value;
+  // Flags that do not make a node accessibilty focusable by themselves.
+  private static final int TRIVIAL_FLAGS =
```

Comment:
they aren't trivial though, right?

Reply:
I can't come up with a better name, how about UNIMPORTANT_FLAGS, UNINTERESTING_FLAGS?

Reply:
another way to look at this is to reverse `~TRIVIAL_FLAGS`, so now it could be named `FOCUSABLE_FLAGS`

Reply:
That works too, was think about how maintainable if people adds more flag, but i think either way is same